owslib/csw.py

```diff
@@ -35,7 +33,7 @@
 from owslib.dif import DIF
 from owslib.gm03 import GM03
 from owslib.namespaces import Namespaces
-from owslib.util import cleanup_namespaces, bind_url, add_namespaces, openURL
+from owslib.util import cleanup_namespaces, bind_url, add_namespaces, OrderedDict, Authentication
```

Comment:
Why we adding `OrderedDict` here?

Reply:
Not sure how that got added. I'll remove it.